Med_Table.py

```diff
@@ -0,0 +1,8 @@
+from select import select
+import sqlite3
+
+conn = sqlite3.connect('Med_Table.db')
+db = conn.cursor()
+
+db.execute('''Create table Compositions (ID TEXT, Composition_name TEXT, Generic_comp_eq TEXT, PRIMARY KEY (ID));''')
+db.execute('''Create table Medicines(med_id TEXT, Medicine_name TEXT, [Manufacturer Comp_ID] TEXT, FOREIGN KEY ([Manufacturer Comp_ID]) REFERENCES Compositions(ID));''')
```

Comment:
Hi @rahul2001dewangan!
Manufacturer and Comp_ID are separate fields.
I updated the issue with the correct requirements. 
Please make the changes accordingly

Reply:
I have made the changes. Please review.